server/controllers/bookmark.controller.js

```diff
@@ -267,7 +267,7 @@ function unbookmark(req, res, next) {
       return Bluebird.all([newFavorite.save()]);
     })
     .then((favorite) => {
-      req.favorite = favorite[0]; // eslint-disable-line no-param-reassign
```

Comment:
The logic here seems to differ from the original?

Reply:
Yeah, you are right. I was mistake this line.

Reply:
It doesn't look like we even use the `req.favorite`?

Reply:
It should be req.favorite = favorite[0];